src/components/FilterDrawer/FilterDrawer.js

```diff
@@ -53,13 +53,13 @@ ResourceTypeFilter.propTypes = {
 
 const FilterDrawer = ({
   toggleResourceTypeFilterAction,
-  orderedResourceTypeFilters,
+  resourceTypeFilters,
   children,
 }) => {
   const renderDrawer = () => (
     <View>
       <Text style={styles.drawerTitle}>Resource Type Filters</Text>
-      {Object.entries(orderedResourceTypeFilters).map(([resourceType, value]) => (
+      {Object.entries(resourceTypeFilters).map(([resourceType, value]) => (
```

Comment:
sorting by label should occur here

Reply:
sorting could occur in the selector, and it returns an Array of Objects, maybe:
`[{ type, label, enabled }, { type, label, enabled }, { type, label, enabled }]`